Sources/BraveNews/FeedItemView.swift

```diff
@@ -365,6 +365,46 @@ extension FeedItemView {
         ]
       )
     )
+    
+    /// Defines a feed item layout where the logo resides on top taking up
+    /// the full-width then underneath is a padded label stack with the title,
+    /// description, space and finally the button
+    ///
+    /// ```
+    /// ╭─────────────────╮
+    /// │                 │
+    /// │      logo       │
+    /// │                 │
+    /// ├─────────────────┤
+    /// │ title           │
+    /// │ descriotion     │
+    /// ├─────────────────┤
+    /// ├─────────────────┤
+    /// │ action button   │
+    /// ╰─────────────────╯
+    /// ```
+    ///
+    static let rateCard = Layout(
+      root: .init(
+        axis: .vertical,
+        children: [
+          .thumbnail(.aspectRatio(1.5)),
```

Comment:
Can we also fix the thumbnail? In the screenshots/video its cutoff

Reply:
Design has updated using Android Design Fix at https://github.com/brave/brave-ios/pull/7589/commits/8f5682712a33dc7f60d1c39b87067dc65c51458e